client/homebrew/brewRenderer/toolBar/toolBar.jsx

```diff
@@ -0,0 +1,124 @@
+require('./toolBar.less');
+const React = require('react');
+const { useState, useEffect } = React;
+const _ = require('lodash')
+
+const MAX_ZOOM = 300;
+const MIN_ZOOM = 10;
+
+const ToolBar = ({ onZoomChange, currentPage, onPageChange, totalPages })=>{
+
+	const [zoomLevel, setZoomLevel] = useState(100);
+	const [pageInput, setPageInput] = useState(currentPage);
+
+	useEffect(()=>{
+		onZoomChange(zoomLevel);
+	}, [zoomLevel]);
+
+	useEffect(()=>{
+		setPageInput(currentPage);
+	}, [currentPage])
+
+	const handleZoomChange = (delta)=>{
```

Comment:
Can we rename 'handleZoomIncrement` to make clear this only adds/subtracts a delta from the zoom?

Reply:
So rename `handleZoomChange`?   Would `handleZoomButton` work?

Reply:
That's fine. Just to clarify its handling a specific type of zoom change, not all zoom change.